lib/core/src/lib/styles/_index.scss

```diff
@@ -29,50 +29,50 @@
         // theme colors
         --theme-primary-color: mat.get-color-from-palette($primary),
         --theme-primary-color-default-contrast: mat.get-color-from-palette($primary, default-contrast),
-        --theme-primary-contrast-500:mat.get-color-from-palette($primary, '500-contrast'),
-        --theme-primary-50:mat.get-color-from-palette($primary, 50),
-        --theme-primary-100:mat.get-color-from-palette($primary, 100),
-        --theme-primary-300:mat.get-color-from-palette($primary, 300),
-        --theme-primary-900:mat.get-color-from-palette($primary, 900),
+        --adf-theme-primary-contrast-500:mat.get-color-from-palette($primary, '500-contrast'),
+        --adf-theme-primary-50:mat.get-color-from-palette($primary, 50),
+        --adf-theme-primary-100:mat.get-color-from-palette($primary, 100),
+        --adf-theme-primary-300:mat.get-color-from-palette($primary, 300),
+        --adf-theme-primary-900:mat.get-color-from-palette($primary, 900),
 
         --theme-warn-color: mat.get-color-from-palette($warn),
         --theme-warn-color-default-contrast: mat.get-color-from-palette($warn, default-contrast),
-        --theme-warn-contrast-500: mat.get-color-from-palette($warn, '500-contrast'),
+        --adf-theme-warn-contrast-500: mat.get-color-from-palette($warn, '500-contrast'),
 
         --theme-accent-color: mat.get-color-from-palette($accent),
         --theme-accent-contrast-500:mat.get-color-from-palette($accent, '500-contrast'),
         --theme-accent-color-a200: mat.get-color-from-palette($accent, A200),
         --theme-accent-color-default-contrast: mat.get-color-from-palette($accent, default-contrast),
         --theme-accent-500:mat.get-color-from-palette($accent, 500),
 
-        --theme-foreground-base-color: mat.get-color-from-palette($foreground, base),
```

Comment:
@tomaszhanaj you essentially destroy all the variables that ACA/ADW and customers are setting from the applications

Reply:
@tomaszhanaj can you please refer to the ACA as an example why renaming all this vars breaks everything https://github.com/Alfresco/alfresco-content-app/blob/common-content-services-migration/projects/aca-content/src/lib/ui/variables/variables.scss#L51